PatchesApi.Tests/V1/Boundary/Validation/GetByIdRequestValidatorTests.cs

```diff
@@ -0,0 +1,37 @@
+using FluentValidation.TestHelper;
+using PatchesApi.V1.Boundary.Request;
+using PatchesApi.V1.Boundary.Request.Validation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PatchesApi.Tests.V1.Boundary.Validation
+{
+    public class GetByIdRequestValidatorTests
+    {
+        private readonly GetByIdRequestValidator _classUnderTest;
+
+        public GetByIdRequestValidatorTests()
+        {
+            _classUnderTest = new GetByIdRequestValidator();
+        }
+
+        [Fact]
+        public void RequestShouldErrorWithNullId()
+        {
+            var query = new PatchesQueryObject();
```

Comment:
Perhaps explicitly set the Id to null

Reply:
I don't think we can do that as Id is a Guid value so I can't set it to null